web/src/main.jsx

```diff
@@ -4,14 +4,17 @@ import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import './index.css';
 import Home from './home/Home.jsx';
 import Graph from './graph/Graph.jsx';
+import { WebSocketProvider } from './components/WebSocketContext';
```

Comment:
Fix the import syntax
```suggestion
import { WebSocketProvider } from './components/websocketContext';
```